packages/ws-worker/src/start.ts

```diff
@@ -1,124 +1,11 @@
-  WORKER_PORT,
-  WORKER_REPO_DIR,
-  WORKER_SECRET,
-  WORKER_STATE_PROPS_TO_REMOVE,
-} = process.env;
-
-const args = yargs(hideBin(process.argv))
-  .command('server', 'Start a ws-worker server')
-  .option('port', {
-    alias: 'p',
-    description: 'Port to run the server on. Env: WORKER_PORT',
-    type: 'number',
-    default: WORKER_PORT || 2222,
-  })
-  // TODO maybe this is positional and required?
-  // frees up -l for the log
-  .option('lightning', {
-    alias: ['l', 'lightning-service-url'],
-    description:
-      'Base url to Lightning websocket endpoint, eg, ws://localhost:4000/worker. Set to "mock" to use the default mock server. Env: WORKER_LIGHTNING_SERVICE_URL',
-    default: WORKER_LIGHTNING_SERVICE_URL || 'ws://localhost:4000/worker',
-  })
-  .option('repo-dir', {
-    alias: 'd',
-    description:
-      'Path to the runtime repo (where modules will be installed). Env: WORKER_REPO_DIR',
-    default: WORKER_REPO_DIR,
-  })
-  .option('secret', {
-    alias: 's',
-    description:
-      'Worker secret. (comes from WORKER_SECRET by default). Env: WORKER_SECRET',
-    default: WORKER_SECRET,
-  })
-  .option('lightning-public-key', {
-    description:
-      'Base64-encoded public key. Used to verify run tokens. Env: WORKER_LIGHTNING_PUBLIC_KEY',
-    default: WORKER_LIGHTNING_PUBLIC_KEY,
-  })
-  .option('log', {
-    description:
-      'Set the log level for stdout (default to info, set to debug for verbose output). Env: WORKER_LOG_LEVEL',
-    default: WORKER_LOG_LEVEL || 'debug',
-    type: 'string',
-  })
-  .option('loop', {
-    description: 'Disable the claims loop',
-    default: true,
-    type: 'boolean',
-  })
-  .option('mock', {
-    description: 'Use a mock runtime engine',
-    default: false,
-    type: 'boolean',
-  })
-  .option('backoff', {
-    description:
-      'Claim backoff rules: min/max (in seconds). Env: WORKER_BACKOFF',
-    default: WORKER_BACKOFF || '1/10',
-  })
-  .option('capacity', {
-    description: 'max concurrent workers. Env: WORKER_CAPACITY',
-    default: WORKER_CAPACITY ? parseInt(WORKER_CAPACITY) : 5,
-    type: 'number',
-  })
-  .option('state-props-to-remove', {
-    description:
-      'A list of properties to remove from the final state returned by a job. Env: WORKER_STATE_PROPS_TO_REMOVE',
-    default: WORKER_STATE_PROPS_TO_REMOVE ?? ['configuration', 'response'],
-    type: 'array',
-  })
-  .option('run-memory', {
-    description:
-      'Maximum memory allocated to a single run, in mb. Env: WORKER_MAX_RUN_MEMORY_MB',
-    type: 'number',
-    default: WORKER_MAX_RUN_MEMORY_MB
-      ? parseInt(WORKER_MAX_RUN_MEMORY_MB)
-      : 500,
-  })
-  .option('max-run-duration-seconds', {
-    alias: 't',
-    description:
-      'Default run timeout for the server, in seconds. Env: WORKER_MAX_RUN_DURATION_SECONDS',
-    type: 'number',
-    default: WORKER_MAX_RUN_DURATION_SECONDS || 60 * 5, // 5 minutes
-  })
-  .parse() as Args;
+const args = parseArgs(process.argv)
```

Comment:
`process.argv` is available inside the `parseArgs` function - there's actually no need to pass it as an argument

Reply:
I wanted to clarify that the arguments are passed to the `parseArgs` function so we can run test cases using custom argv values, as you mentioned:
```
const argv = 'node cli.js --port 3000 --log info --max-run-duration-seconds 120'
const args = parseArgs(argv).split(" ");
```
This allows us to simulate different command-line inputs during testing.

Reply:
I see - good point!